### ISSUE TITLE
Multiple tables on the same page

### ISSUE THREAD

**Issue by huyz** [vincjo/svelte-simple-datatables#21]
it looks to me like this only supports one table at a time, right?

**Comment by andreasbhansen** [comment]
Yeah, also reported in #10.

**Comment by tyler-boyd** [comment]
Is there a reason that this library uses a global store instead of a `let:` binding on the slot? 
I think this would be a simple change and I'd be happy to open a PR, but I do get the impression this library isn't actively maintained.

**Comment by vincjo** [comment]
Hi tyler-boyd

Not sure it works with a slot. It uses derived stores to filter the data.

Maybe you know more than I do. The code is located [here](https://github.com/vincjo/svelte-simple-datatables/blob/master/src/stores/data.js)

If you have any suggestions, I'll take it

**Comment by tyler-boyd** [comment]
Thanks @vincjo, that makes sense! I'm fairly new to Svelte so I don't have much expertise, but the way I would've guessed a library like this would work is 

```svelte
<Datatable {settings} {data}>
  <span name="header">...</span>
  <span name="row" let:row>... {row.id} {row.name} (access any fields from a data element) ...</span>
</Datatable>
```

So this would use let: bindings instead of stores. Perhaps there's disadvantages to this approach? Again, fairly new and haven't implemented anything like this with the same level of features as this library (all the sorting/filtering/pagination etc).

**Comment by vincjo** [comment]
Multiple tables on the same page has been released in 0.1.27 / 2021-11
